### DISCUSSION TITLE
How do I get the Spectacle default shortcuts in Rectangle Pro?

### DISCUSSION THREAD

**Discussion by rxhanson** [rxhanson/RectanglePro-Community#181] [Q&A] [answered]
Rectangle has an option to use the Spectacle default shortcuts. Rectangle Pro does not. How do I get them?

**Comment by rxhanson** [comment, marked as answer]
To get Spectacle defaults, you can execute the following terminal command, and restart the app:

```
defaults write com.knollsoft.Hookshot alternateDefaultShortcuts -bool true
```

You can go back to the other set of defaults by setting that to false.

**Reply by faustbrian** [reply]
@rxhanson was this feature removed? I'm on a new machine and this no longer seems to work when using Rectangle (Pro in my case)

**Reply by rxhanson** [reply]
This feature is still in, and is not something that I would remove. Maybe try this Terminal command:

```
defaults write com.knollsoft.Hookshot.plist alternateDefaultShortcuts -bool true
```

And make sure to restart the app afterwards.
